### PR TITLE
Update worldserver.conf

### PR DESCRIPTION
**Pull request by jorge990125**

### CODE REVIEW

File: src/server/worldserver/worldserver.conf.dist [hunk 1]
```diff
@@ -66,8 +66,6 @@ Arena.SeasonReward.Cleanup.Titles = 44 43 45
 Arena.SeasonReward.RatingThreshold.3x3 = -1
 Arena.SeasonReward.RatingThreshold.5x5 = -1
 Arena.SeasonReward.RatingThreshold.2x2 = -1
-Updates.AutoSetup = 1
-Updates.EnableDatabases = 1
```
Comment (leelf00) [inline src/server/worldserver/worldserver.conf.dist:70]: changed to 0

File: src/server/worldserver/worldserver.conf.dist [hunk 2]
```diff
@@ -66,8 +66,6 @@ Arena.SeasonReward.Cleanup.Titles = 44 43 45
 Arena.SeasonReward.RatingThreshold.3x3 = -1
 Arena.SeasonReward.RatingThreshold.5x5 = -1
 Arena.SeasonReward.RatingThreshold.2x2 = -1
-Updates.AutoSetup = 1
```
Comment (leelf00) [inline src/server/worldserver/worldserver.conf.dist:69]: change to 0

Comment (jorge990125) [reply src/server/worldserver/worldserver.conf.dist:69]: it's duplicated


UPDATE SETTINGS
#
#    Updates.EnableDatabases
#        Description: A mask that describes which databases shall be updated.
#
#        Following flags are available
#           DATABASE_LOGIN     = 1, // Auth database
#           DATABASE_CHARACTER = 2, // Character database
#           DATABASE_WORLD     = 4, // World database
#
#        Default:     7  - (All enabled)
#                     4  - (Enable world only)
#                     0  - (All disabled)

Updates.EnableDatabases = 0

#
#    Updates.AutoSetup
#        Description: Auto populate empty databases.
#        Default:     1 - (Enabled)
#                     0 - (Disabled)

Updates.AutoSetup   = 0

**Comment by leelf00** [comment]
Thanks for PR.

**Pull request merged**